app/controllers/users_controller.rb

```diff
@@ -46,7 +46,7 @@ def create
     if Rails.env == "test"
       success = @user && @user.save
     else
-      success = verify_recaptcha(:model => @user, :message => "Please re-enter the words from the image again.") && @user && @user.save
+      success = verify_recaptcha(:model => @user) && @user && @user.save
     end
```

Comment:
Is :message redundant or no longer necessary?

Reply:
it complained during runtime, so I removed, however it should be possible see https://github.com/ambethia/recaptcha#verify_recaptcha